src/test/scala/nl.knaw.dans.easy.deposit/servlets/DepositServletErrorSpec.scala

```diff
@@ -39,6 +41,26 @@ class DepositServletErrorSpec extends TestSupportFixture with ServletFixture wit
   }
   addServlet(depositServlet, "/*")
 
+  "constructor" should "fail with not existing multipart location" in {
+    val props = minimalAppConfig.properties.clone().asInstanceOf[PropertiesConfiguration]
+    props.addProperty("multipart.location", "notExistingLocation")
+    the[ConfigurationException] thrownBy new DepositServlet(new EasyDepositApiApp(new Configuration("", props))) should
+      have message s"Configuration error: ${ File("notExistingLocation").path.toAbsolutePath } not found or not a directory"
+  }
+
+  it should "fail with empty threshold value" in {
+    val props = minimalAppConfig.properties.clone().asInstanceOf[PropertiesConfiguration]
+    props.addProperty("multipart.file-size-threshold", "")
+    the[ConversionException] thrownBy new DepositServlet(new EasyDepositApiApp(new Configuration("", props))) should
+      have message s"'multipart.file-size-threshold' doesn't map to an Integer object"
+  }
+
+  it should "succeed without threshold property" in {
```

Comment:
Or do we want to make the property mandatory to avoid heap space errors?

Reply:
YES PLEASE!!! make at least the threshold mandatory.
I'm tempted to even request the location to be mandatory, but I'm not yet sure about that. What would be the consequences of **not** making the location mandatory and **not** providing it either?